src/beta_inc.jl

```diff
@@ -825,3 +825,120 @@ function beta_inc(a::T, b::T, x::T, y::T) where {T<:Union{Float16, Float32}}
 end
 beta_inc(a::Real, b::Real, x::Real, y::Real) = beta_inc(promote(float(a), float(b), float(x), float(y))...)
 beta_inc(a::T, b::T, x::T, y::T) where {T<:AbstractFloat} = throw(MethodError(beta_inc,(a, b, x, y,"")))
+
+#GW Cran, KJ Martin, GE Thomas, Remark AS R19 and Algorithm AS 109: A Remark on Algorithms AS 63: The Incomplete Beta Integral and AS 64: Inverse of the Incomplete Beta Integeral,
+#Applied Statistics,
+#Volume 26, Number 1, 1977, pages 111-114. 
+
+"""
+    beta_inc_inv(a,b,p,q,lb=logbeta(a,b)[1])
+
+Computes inverse of incomplete beta function. Given `a`,`b` and ``I_{x}(a,b) = p`` find `x` and return tuple (x,y).
+"""
+function beta_inc_inv(a::Float64, b::Float64, p::Float64, q::Float64; lb = logbeta(a,b)[1])
+    fpu = 1e-30
+    x = p
+    if p == 0.0
+        return (0.0, 1.0)
+    elseif p == 1.0
+        return (1.0, 0.0)
+    end
+
+    #change tail if necessary
+
+    if p > 0.5
+        aa = q
+        pp = b
+        qq = a
+        indx = true
+    else
+        aa = p
+        pp = a
+        qq = b
+        indx = false
+    end
+
+    #Initial approx
+
+    r = sqrt(-log(aa^2))
+    y = r - @horner(r, 2.30753e+00, 0.27061e+00) / @horner(r, 1.0, .99229e+00, .04481e+00)
+
+    if a > 1.0 && b > 1.0
+        r = (y^2 - 3.0)/6.0
+        s = 1.0/(2*pp - 1.0)
+        t = 1.0/(2*qq - 1.0)
+        h = 2.0/(s+t)
+        w = y*sqrt(h+r)/h - (t-s)*(r + 5.0/6.0 - 2.0/(3.0*h))
+        x = pp/ (pp+qq*exp(w^2))
+    else
+        r = 2.0*qq
+        t = 1.0/(9.0*qq)
+        t = r*(1.0-t+y*sqrt(t))^3
+        if t <= 0.0
+            x = -expm1((log((1.0-aa)*qq)+lb)/qq)
+        else
+            t = (4.0*pp+r-2.0)/t
+            if t <= 1.0
+                x = exp((log(aa*pp)+lb)/pp)
+            else
+                x = 1.0 - 2.0/(t+1.0)
+            end
+        end
+    end
+
+    #solve x using modified newton-raphson iteration
+
+    r = 1.0 - pp
+    t = 1.0 - qq
+    yprev = 0.0
```

Comment:
similarly, `y`/`yprev` from here down should be named something related to `p`/`pp`

Maybe `pp_approx`/`pp_approx_prev`?

(I don't think the `y` above here is related)

Reply:
Yes, sure!